### ISSUE TITLE
[bug]: 

### ISSUE THREAD

**Issue by wamaw123** [makeplane/plane#2335]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

It is either that labels are limited to each project / sub-project or I can't access labels in my workspace and need to recreate them for different issues.

### Steps to reproduce

Set label for an Issue
Try to set the same label for another issue
Impossible to see the first label even when reloading the workspace : either the labels are restricted to the project (which is very limiting) and thus comon labels must exist in multiple versions, or there is issue to access the labels globally in the workspace.

### Browser

Google Chrome

### Version

Cloud

**Comment by tkjaergaard** [comment]
I agree with @tim-the-arcane on this. We also want to take advantage of the views feature. But with every project having their own labels, we continuously need to update the view, and select the labels.

Maybe at least group the labels that are the same into a single label for the views. So if you have a label called "billing" across multiple projects, group them into a single "billing" label for the views filer. Then backed by templates it could work.

**Comment by vihar** [comment]
Labels are project-scoped today. Workspace-level governance, which is where shared labels belong, will be part of the Enterprise Grid plan.

Closing this out.
